pkg/controller/fileintegrity/fileintegrity_controller.go

```diff
@@ -372,11 +559,11 @@ func masterAideDaemonset() *appsv1.DaemonSet {
 							},
 						},
 						{
-							Name: "aide-script",
+							Name: aideScriptConfigMapName,
 							VolumeSource: corev1.VolumeSource{
 								ConfigMap: &corev1.ConfigMapVolumeSource{
 									LocalObjectReference: corev1.LocalObjectReference{
-										Name: "aide-script",
+										Name: aideScriptConfigMapName,
```

Comment:
You might want to do the same change in `workerAideDaemonSet`

Reply:
Nah, sorry, wrong place. What I mean is that you might want to squash this change to your PR:
```

diff --git a/pkg/controller/fileintegrity/fileintegrity_controller.go b/pkg/controller/fileintegrity/fileintegrity_controller.go                                                                                                                                                                                             
index 7d31f01..a5849d8 100644                                                                                                                                                                                                                                                                                                
--- a/pkg/controller/fileintegrity/fileintegrity_controller.go                                                                                                                                                                                                                                                               
+++ b/pkg/controller/fileintegrity/fileintegrity_controller.go                                                                                                                                                                                                                                                               
@@ -530,7 +530,7 @@ func masterAideDaemonset() *appsv1.DaemonSet {                                                                                                                                                                                                                                                           
                                                        VolumeSource: corev1.VolumeSource{
                                                                ConfigMap: &corev1.ConfigMapVolumeSource{
                                                                        LocalObjectReference: corev1.LocalObjectReference{
-                                                                               Name: "aide-conf",                                                                                                                                                                                                                           
+                                                                               Name: defaultConfigMapName,                                                                                                                                                                                                                  
                                                                        },
                                                                },
                                                        }
```

Reply:
good catch, thanks!